bartender/app.py

```diff
@@ -100,14 +100,10 @@ def __init__(self):
             ),
         ]
 
-        # Only want to run the MongoPruner if it would do anything
         tasks, run_every = self._setup_pruning_tasks()
-        if run_every:
-            self.helper_threads.append(
-                HelperThread(
-                    MongoPruner, tasks=tasks, run_every=timedelta(minutes=run_every)
-                )
-            )
+        self.helper_threads.append(
+            HelperThread(MongoPruner, tasks=tasks, run_every=timedelta(seconds=10))
+        )
```

Comment:
Did you mean to ignore `run_every`?

Reply:
No, this is a mistake. I was testing locally.